src/wrapperUpdater.ts

```diff
@@ -67,7 +67,7 @@ class WrapperUpdater implements IWrapperUpdater {
     }
 
     const {exitCode, stderr} = await cmd.execWithOutput(
-      'gradle',
+      './gradlew',
```

Comment:
I'm not sure were `gradle` comes from but I guess it is from the Github Actions environment. This version might not support newer features of gradle files and thus fail the action. Using the `gradlew` ensures that updates work also with RC features from one RC version to the next.

Reply:
That's interesting! Yes, the `gradle` executable here comes from the action environment and is normally quite [up to date](https://github.com/actions/virtual-environments/blob/37a897095f6952ce45e49011084fe9efaa24e8e6/images/linux/scripts/installers/java-tools.sh#L70).

The reason why I originally decided to use the system binary rather than the project `gradlew` is because if the latter has been compromised in your project it could potentially download a newer/fake updated binary.

What specific error have you encountered when updating between RC versions?

Reply:
This behaviour has actually been changed as part of https://github.com/gradle-update/update-gradle-wrapper-action/pull/234, so currently we use  `./gradlew`.